### PR TITLE
[docs] Customer.LoyaltyInfo, Restriction fix

### PR DESCRIPTION
**Pull request by ondrahermanek**
- Fixed bug about restrictions - invalid value for `states` for closed

- Added `LoyaltyInfo` to `Customer`

### CODE REVIEW

File: mews-operations/reservations.md [hunk 1]
```diff
@@ -157,7 +157,11 @@ The third `reservation` definition shows the partial cancellation - cancelling t
                     "nationalityCode": "US",
                     "languageCode": "en-US",
                     "telephone": "1-369-81891",
-                    "loyaltyCode": "PG60972345"
+                    "loyaltyCode": "PG60972345",
+                    "loyaltyInfo": {
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:161]: If possible, can we use real (but anonymized) examples for LoyaltyInfo fields.

File: mews-operations/reservations.md [hunk 2]
```diff
@@ -263,10 +267,18 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | `title` | `string` [Title](#title) | optional | Customer title, e.g. "Mister" |
 | `telephone` | `string` | optional | Telephone. |
 | `loyaltyCode` | `string` | optional | Loyalty code of the customer. Processed only within main customer. |
+| `loyaltyInfo` | [`LoyaltyInfo`](#loyalty-info) object | optional | Loyalty information of the customer. Processed only within main customer or [`Availability block`](../channel-manager-operations/availabilityBlock.md#availability-block) booker. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:270]:
```suggestion
| `loyaltyInfo` | [`Loyalty Info`](#loyalty-info) object | optional | Loyalty information of the customer. Processed only within main customer or [`Availability block`](../channel-manager-operations/availabilityBlock.md#availability-block) booker. |
```

File: mews-operations/reservations.md [hunk 3]
```diff
@@ -263,10 +267,18 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
 | `nationalityCode` | `string` | optional | [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) - two letter country code or [ISO 3166-1 alpha-3 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) - three letter country code. |
 | `languageCode` | `string` | optional | Language [code](https://msdn.microsoft.com/en-us/library/ee825488) of the communication language of the customer. This language will be used as the default for communication with the customer. |
 | `address` | [`Address`](configuration.md#address) object | optional | Represents address. |
 
+#### Loyalty Info
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `membershipId` | `string` | required | Loyalty memebership identifier of the Customer. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:279]:
```suggestion
| `membershipId` | `string` | required | Loyalty membership identifier of the Customer. |
```

File: mews-operations/reservations.md (hunk 2 above)
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:270]: I'm not sure what is meant by "Processed only within main customer or Availability Block booker", is it possible to put this into different words?

**Review by MikeAdamsMews — COMMENTED** [review]
@ondrahermanek just a couple of suggested changes. In addition, can you please create a **Changelog** entry for this, following the standard format? Thanks.

File: mews-operations/reservations.md (hunk 2 above)
Comment (ondrahermanek) [reply mews-operations/reservations.md:270]: it is copied from above, but sure, we can change it.

it is supposed so say, that it is taken/send only for the main customer/block booker. for other `customer` profiles in that message we ignore the value (even if provided)

File: mews-operations/reservations.md (hunk 1 above)
Comment (ondrahermanek) [reply mews-operations/reservations.md:161]: yep, why not, do you know some? 😄

**Review by patrikmiksovsky95 — APPROVED** [review]

File: mews-operations/reservations.md (hunk 2 above)
Comment (MikeAdamsMews) [reply mews-operations/reservations.md]: Ah, thanks for the explanation @ondrahermanek. In that case, I would suggest the following very small change:

**Loyalty code:**
Loyalty code of the customer. Processed only for the main customer.

**Loyalty info:**
Loyalty information of the customer. Processed only for the main customer or Availability block booker.

**Comment by MikeAdamsMews** [comment]
@ondrahermanek can you add a changelog, then we're all done and good to go!

## 11th April 2024

* Added loyalty info to Request in [Mews API Operations: Process Group](../mews-operations/reservations.md#process-group).

**Pull request merged**